src/orion/algo/mofa/sampler.py

```diff
@@ -15,44 +22,71 @@ def __init__(self):
         return
 
     @staticmethod
-    def normalized_olh_space(roi_space):
-        """
-        Generates a normalized parameter space for the orthogonal Latin hypercube
-
-        Parameters
-        ----------
-        roi_space: `orion.algo.space.Space`
-            Parameter space region-of-interest
+    def _is_prime(x):
+        if x < 2 or (x % 2 == 0 and x > 2):
+            return False
+        for i in range(3, int(np.sqrt(x - 1)) + 1, 2):
+            if x % i == 0:
+                return False
+        return True
 
-        Returns
-        -------
-        A `skopt.space.Space` instance with a normalized parameter space
-        """
-        norm_space = []
-        for _ in roi_space.items():
-            norm_space.append((0.0, 1.0))
-        return SkSpace(norm_space)
+    @staticmethod
+    def _find_n_levels(d):
+        x = d - 1
+        while True:
+            if Sampler._is_prime(x):
+                return x
+            else:
+                x = x + 1
```

Comment:
```suggestion
        while not Sampler._is_prime(x):
            x = x + 1
```
Sorry if I'm a bit OCD here. :grimacing: